prow/overlays/cnv-prod/plugins.yaml

```diff
@@ -87,6 +87,7 @@ approve:
       - thoth-station
     require_self_approval: true
     ignore_review_state: false
+    lgtm_acts_as_approve: false
```

Comment:
@HumairAK @anishasthana @4n4nd we want this for operate-first as well! :smile:

Reply:
you got it, if we merge it ;)